changelog.json

```
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "3.2.0": [
+      "[New] Preview your pull request - view a diff of changes in your current branch.",
+      "[Added] Add Tabby terminal integration for macOS - #16040. Thanks @ansidev!",
+      "[Added] Add JetBrains DataSpell support on Windows - #16020. Thanks @tsvetilian-ty!",
+      "[Added] Add JetBrains DataSpell support on macOS- #16020. Thanks @jm-rivera!",
+      "[Added] Editors installed with JetBrains Toolbox supported on Linux - #16029. Thanks @adil192!",
+      "[Added] Support for numerous additional editors on Linux - #16030. Thanks @shiftkey!",
+      "[Fixed] Include renamed files in the commit summary changed files tooltip - #15155. Thanks @haykam821!",
+      "[Improved] The context menu for a file in the changed files list can be invoked by keyboard shortcuts - #16114",
+      "[Improved] The file status of a changed file is announced with voice over - #16098",
+      "[Improved] The changes list announces file selection position with voice over - #16108",
```

`VoiceOver` is a macOS thing, which I think is called `Narrator` on Windows. I think the generic term is `screen reader`.
```suggestion
      "[Improved] The file status of a changed file is announced by screen readers - #16098",
      "[Improved] The changes list announces file selection position with screen readers - #16108",
```

---

Good call out.